packages/wdio-cli/src/templates/afterTest.ejs

```diff
@@ -0,0 +1,14 @@
+/**
+     * Function to be executed after a test (in Mocha/Jasmine) or a step (in Cucumber) starts.
+     * @param {Object} test test details
+     */<% 
+if (reporters.length && reporters.includes('allure')) {%>
+    afterTest: function(test) {
+        if (test.error !== undefined) {
+          browser.takeScreenshot();
```

Comment:
```suggestion
            browser.takeScreenshot();
```

Reply:
Ah I missed that one, just pushed the fix.